ngrinder-controller/src/test/java/org/ngrinder/script/service/GitHubFileEntryServiceTest.java

```diff
@@ -0,0 +1,88 @@
+package org.ngrinder.script.service;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.ngrinder.AbstractNGrinderTransactionalTest;
+import org.ngrinder.common.util.CompressionUtils;
+import org.ngrinder.model.User;
+import org.ngrinder.script.model.FileEntry;
+import org.ngrinder.script.model.GitHubConfig;
+import org.ngrinder.script.repository.MockFileEntityRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+
+public class GitHubFileEntryServiceTest extends AbstractNGrinderTransactionalTest {
+
+    private static final String GITHUB_CONFIG_NAME = ".gitconfig.yml";
+
+    @Autowired
+    private GitHubFileEntryService gitHubFileEntryService;
+
+    @Autowired
+    private MockFileEntityRepository mockFileEntityRepository;
+
+    @Before
+    public void before() throws IOException {
+        File file = new File(System.getProperty("java.io.tmpdir"), "repo");
```

Comment:
How about use [Junit TempDir](https://www.baeldung.com/junit-5-temporary-directory#2-tempdir-on-an-instance-field)?
It will be deleted automatically after test

Reply:
We are using junit 4.13.1 😭. And TempDir seems to be supported since Junit5.
https://github.com/naver/ngrinder/blob/f6d067271ad10716fe4f75ba77216a9fc7143306/gradle.properties#L4